ansible/library/generate_golden_config_db.py

```diff
@@ -344,6 +344,24 @@ def update_dns_config(self, config):
         else:
             return config
 
+    def generate_ft2_golden_config_db(self):
+        """
+        Generate golden_config for FT2 to enable FEC.
+        **Only PORT table is updated**.
+        """
+        SUPPORTED_TOPO = ["ft2-64"]
+        if self.topo_name not in SUPPORTED_TOPO:
+            return "{}"
+        SUPPORTED_PORT_SPEED = ["100000", "200000", "400000", "800000"]
+        ori_config = json.loads(self.get_config_from_minigraph())
+        port_config = ori_config.get("PORT", {})
+        for name, config in port_config.items():
+            # Enable FEC for ports with supported speed
+            if config["speed"] in SUPPORTED_PORT_SPEED and "fec" not in config:
+                config["fec"] = "rs"
```

Comment:
make rs parameterable since fec can be fc as well or explicit none

Reply:
That's a good idea.
But currently the infrastructure doesn't support passing argument to `generate_golden_config_db` library. I would like to do the infra enhancement in a separate PR. 
For topo `ft2-64`, the scenario is limited. So hardcoding `rs` should be good.